roles/nodeexporter/defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+# Copyright 2023 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+node_exporter_directory: /etc/node_exporter
+node_exporter_tarball_url: https://github.com/prometheus/node_exporter/releases/download/v1.7.0/node_exporter-1.7.0.linux-amd64.tar.gz
+node_exporter_tarball_file: node_exporter.tar.gz
+node_exporter_service_directory: /etc/systemd/system/node_exporter.service
+
+prometheus_user : prometheus
```

Comment:
It might be better to have a separate user/group for node exporter than that used for Prometheus.

Reply:
Replaced